src/Drivers/LocalFileSystem.ts

```diff
@@ -32,57 +33,65 @@ export class LocalFileSystem extends Storage {
 		location: string,
 		content: Buffer | Stream | string,
 		options?: fs.WriteFileOptions
-	): Promise<boolean> {
+	): Promise<Response> {
 		if (isReadableStream(content)) {
 			return this.put(location, content, Object.assign({ flags: 'a' }, options));
+		} else {
+			const result = await fs.appendFile(this._fullPath(location), content, options);
+			return { raw: result };
 		}
-
-		await fs.appendFile(this._fullPath(location), content, options);
-
-		return true;
 	}
 
 	/**
 	 * Copy a file to a location.
 	 */
-	public async copy(src: string, dest: string, options?: fs.CopyOptions): Promise<boolean> {
-		await fs.copy(this._fullPath(src), this._fullPath(dest), options);
+	public async copy(src: string, dest: string, options?: fs.CopyOptions): Promise<Response> {
+		const result = await fs.copy(this._fullPath(src), this._fullPath(dest), options);
 
-		return true;
+		return { raw: result };
 	}
 
 	/**
 	 * Delete existing file.
 	 * This method will not throw an exception if file doesn't exists.
 	 */
-	public async delete(location: string): Promise<boolean> {
-		await fs.remove(this._fullPath(location));
+	public async delete(location: string): Promise<Response> {
+		const result = await fs.remove(this._fullPath(location));
 
-		return true;
+		return { raw: result };
 	}
 
 	/**
 	 * Returns the driver.
 	 */
-	public driver() {
+	public driver(): typeof fs {
 		return fs;
 	}
 
 	/**
 	 * Determines if a file or folder already exists.
 	 */
-	public exists(location: string) {
-		return fs.pathExists(this._fullPath(location));
+	public async exists(location: string): Promise<ExistsResponse> {
+		const result = await fs.pathExists(this._fullPath(location));
+		return { exists: true, raw: result };
 	}
 
 	/**
 	 * Returns the file contents.
 	 */
-	public async get(location: string, encoding = 'utf8'): Promise<Buffer | string> {
+	get(location: string, encoding?: string): Promise<ContentResponse<string>>;
+	get(location: string, encoding: null): Promise<ContentResponse<Buffer>>;
+	public async get(location: string, encoding: string | null = 'utf8'): Promise<ContentResponse<Buffer | string>> {
```

Comment:
another option is to return a Buffer by default if no encoding is passed, like the `fs` API does